.pre-commit-config.yaml

```
@@ -141,3 +141,20 @@ repos:
           '--disable',
           'dangling-hyphen,line-too-long',
         ]
+  - repo: https://github.com/golangci/golangci-lint
+    rev: v1.59.0
+    hooks:
+      # no built-in support for multiple go.mod
+      # https://github.com/golangci/golangci-lint/issues/828
+      - id: golangci-lint
+        name: golangci-lint-arrow
```

How about using `golangci-lint-full` because `golangci-lint run --fix` is the entry used by `golangci-lint-full`?
https://github.com/golangci/golangci-lint/blob/master/.pre-commit-hooks.yaml#L12

```suggestion
      - id: golangci-lint-full
        name: golangci-lint-full-arrow
```